### PR TITLE
Sync master

### PR DESCRIPTION
**Pull request by helper-beep**
I made un update to color_trick.bat, so that there would be a warning, Thanks!!

### CODE REVIEW

**Comment by helper-beep** [comment]
You can see helper-beep/batch@45dd643
Closing for now.

**Comment by double-beep** [comment]
So, now what I will do is to reopen the pull request...,

**Comment by double-beep** [comment]
...approve your excellent changes,...

File: color_trick.bat [hunk 1]
```diff
@@ -1,4 +1,6 @@
 @echo off
+echo This is a color trick!!
+pause
 :TOP
```
Comment (double-beep) [inline color_trick.bat]: Maybe we can ask admin uac with "runas" command or
to select **ONLY** some selections with choice /c

**Comment by double-beep** [comment]
...merge pull request and...

**Comment by double-beep** [comment]
...add "fix" label.
Thanks!

**Comment by double-beep** [comment]
- [ ] Sync master.

**Pull request merged**